datahub-web-react/src/app/entity/dataset/DatasetEntity.tsx

```diff
@@ -68,6 +70,8 @@ export class DatasetEntity implements Entity<Dataset> {
 
     isSearchEnabled = () => true;
 
+    isAccessManagementEnabled = useIsShowAccessManagementEnabled;
```

Comment:
let's get rid of this

Reply:
recieving below error 
"React Hook "useIsShowAccessManagementEnabled" is called in function "visible" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter  react-hooks/rules-of-hooks"

to resolve that we used isAccessManagementEnabled

Reply:
done update

Reply:
(please remove this line)

Reply:
checking w my colleague @chriscollins3456